optimize/util/optimize-commons/src/main/java/io/camunda/optimize/dto/optimize/query/ui_configuration/UIConfigurationResponseDto.java

```diff
@@ -37,6 +37,8 @@ public class UIConfigurationResponseDto {
   private int maxNumDataSourcesForReport;
   private Integer exportCsvLimit;
   private DatabaseType optimizeDatabase;
+  private boolean validLicense;
+  private String environment;
```

Comment:
I know we didn't on the other PR, so maybe this question is broader than Optimize here - but why did we not have an enum for the environments?

Reply:
I added a License enum to both Optimize and the other part of the monorepo. It makes sense

Reply:
Adding a new `enum` caused build issues with finding the new class. I put the new `enum` inside `CamundaLicense` to fix the build issue

Reply:
I was able to figure out and solve the build issues, the `enum` is back to being it own class